test/integration/http.go

```diff
@@ -59,6 +59,11 @@ func (r *http) makeRequests() error {
 	funcs := make(map[string]func() status)
 	for _, src := range testPods {
 		for _, dst := range testPods {
+			if src == "t" && dst == "t" {
+				// this is flaky in minikube
+				continue
+			}
+
```

Comment:
It slows down minikube tests run a lot and do not provide anything useful. There is no proxy involved in t--> t communication.

I am all for deterministic tests.